misc/git/hooks/visitorgen

```diff
@@ -15,11 +15,4 @@
 
 # this script, which should run before committing code, makes sure that the visitor is re-generated when the ast changes
 
-make visitor REWRITER=tmp_rewriter.go
-if ! cmp -s "tmp_rewriter.go" "go/vt/sqlparser/rewriter.go"; then
-	echo "The ast.go has changed, but not rewriter.go"
-	echo "You should 'make visitor' to update the generated rewriter"
-	rm -f tmp_rewriter.go
-	exit 1
-fi
-rm -f tmp_rewriter.go
\ No newline at end of file
+go run ./go/vt/sqlparser/visitorgen/main -compareOnly=true -input=go/vt/sqlparser/ast.go -output=go/vt/sqlparser/rewriter.go
```

Comment:
Instead of (or in addition to?) a precommit hook, what about adding this to the list of tests in `./test/config.json` so that when somebody forgets to regenerate the rewriter it blocks merging the commit on github?

Reply:
Both, and one depending on the other so they always agree is what I would vote for